src/bolin/src/bolin_lidar/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>bolin_lidar</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="eduardopontobarreto@gmail.com">eduardo-barreto</maintainer>
+  <license>TODO: License declaration</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+
+  <build_depend>rclpy</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
```

Comment:
aqui é build_depend ou exec_depend? Pq ser for build tá errado nos outros pacotes